R/dataset.R

```diff
@@ -137,7 +137,23 @@ Dataset.data <- function(id, filters, row.names = NULL, env = solvebio:::.solveE
 
     tryCatch({
         res <- .request('POST', path=path, body=body, env=env)
-        return(formatSolveBioQueryResponse(id, res, row.names=row.names))
+        if (!is.null(col.names)) {
```

Comment:
Doesn't this all need to happen after flattening the response?

Reply:
Yes, will fix that.